ProjectSettings/ProjectVersion.txt

```diff
@@ -0,0 +1 @@
+m_EditorVersion: 5.6.3p4
```

Comment:
Please revert.

Reply:
I still disagree with this, but I'll revert so it doesn't become a blocker.  I wanna try to get the release out before the FCU launch.